blog/_posts/2020-06-03-systematics-blogpost.md

```diff
@@ -0,0 +1,335 @@
+
+When I used this process on the tree with mutations but with no pressure for diversity, I would expect to see values in the 50th percentile range. However, if repeated for the tree with a pressure for diversity, I would then expect values near the 100th percentile. 
+
+## **Results**
+
+**The following table contains the percentile values for a tree with _no_ pressure for diversity**
+
+|          |    |    |    |    |    |    |    |    |    |    | Average | Standard Deviation |
+|----------|----|----|----|----|----|----|----|----|----|----|---------|--------------------|
+| 10 gens  | 61 | 37 | 61 | 97 | 61 | 61 | 61 | 61 | 37 | 37 |    57.4 |        17.93320942 |
+| 20 gens  | 88 | 95 | 99 | 19 | 95 | 88 | 40 | 62 | 95 | 88 |    76.9 |        27.44064301 |
+| 30 gens  | 16 | 95 | 97 | 36 | 98 | 95 | 87 | 36 | 75 | 57 |    69.2 |        30.67318771 |
+| 40 gens  | 11 | 90 | 68 | 11 | 94 | 81 | 90 | 11 | 50 | 11 |    51.7 |        37.23215456 |
+| 50 gens  |  6 | 74 | 58 | 58 | 97 | 39 | 93 | 39 | 74 | 20 |    55.8 |        29.96219841 |
+| 60 gens  | 16 | 90 | 31 | 68 | 50 | 31 | 98 | 68 | 50 | 68 |      57 |        26.38181192 |
+| 70 gens  | 26 | 93 | 26 | 26 | 43 | 61 | 93 | 13 | 61 | 13 |    45.5 |        30.28109054 |
+| 80 gens  | 69 | 81 | 20 | 36 | 69 | 69 | 98 |  9 | 69 | 20 |      54 |        30.23243292 |
+| 90 gens  | 17 | 61 | 17 | 31 | 75 | 61 | 96 | 47 | 61 | 47 |    51.3 |        25.04240847 |
+| 100 gens | 12 | 70 |  1 | 39 | 39 | 56 | 98 | 25 | 70 | 39 |    44.9 |        29.27437256 |
+
+
+**The table below contains the percentile values for a tree _with_ pressure for diversity.**
+|          |     |    |     |     |     |     |     |     |     |     | Average | Standard Deviation |
+|----------|-----|----|-----|-----|-----|-----|-----|-----|-----|-----|---------|--------------------|
+| 10 gens  |  37 | 80 |  80 |  61 |  99 |  61 | 98  | 91  | 61  |  91 |    75.9 |         20.2509259 |
+| 20 gens  |  62 | 77 |  98 |  95 |  99 |  88 | 99  | 77  | 77  |  98 |      87 |        12.99572579 |
+| 30 gens  |  57 | 95 |  98 |  98 |  97 |  97 | 99  | 95  | 95  |  97 |    92.8 |        12.65613597 |
+| 40 gens  | 100 | 94 |  99 |  90 |  97 |  97 | 100 | 97  | 97  | 100 |    97.1 |        3.142893218 |
+| 50 gens  |  97 | 74 | 100 |  93 | 100 |  99 | 93  | 97  | 99  | 100 |    95.2 |        7.913420387 |
+| 60 gens  |  99 | 95 |  95 | 100 |  99 |  98 | 98  | 95  | 99  | 100 |    97.8 |        2.043961296 |
+| 70 gens  |  99 | 99 |  93 | 100 |  99 |  99 | 99  | 99  | 99  | 100 |    98.6 |        2.011080417 |
+| 80 gens  |  99 | 95 |  99 |  98 |  99 | 100 | 81  | 100 | 100 |  99 |      97 |        5.811865258 |
+| 90 gens  |  99 | 96 |  99 |  99 |  98 |  98 | 86  | 99  | 99  |  99 |    97.2 |        4.049691346 |
+| 100 gens |  99 | 99 |  99 |  99 |  98 |  99 | 94  | 99  | 99  |  99 |    98.4 |        1.577621275 |
+
+![Percentile Graph](/assets/BlogImages/PercentileGraph.jpg)
```

Comment:
Great graph!

Reply:
Add a note saying what the error bars are showing (it's standard deviation, right?)